ansible/cloud_providers/vmc_destroy_env.yml

```diff
@@ -0,0 +1,37 @@
+---
+- import_playbook: ../include_vars.yml
+
+- name: Delete Infrastructure
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tasks:
+    - name: Install community.vmware collection for vmware modules
+      command: >-
+        ansible-galaxy collection install
+        -p {{ playbook_dir }}/../collections community.vmware
+        --force-with-deps
```

Comment:
mhh  we should probably have this setup differently.
Requirement files should be used with pinned version.

requirements-vmc.yml?